### PR TITLE
Recipe for ILB based service

### PR DESCRIPTION
**Pull request by kishorerj**
This recipe is for an Service exposing an Internal Load balancer with a fixed IP address. In a shared VPC architecture these Internal load balancer can facilitate service to service communication across service projects in a VPC network.

### CODE REVIEW

**Comment by kishorerj** [comment]
@danielmarzini this is PR for the ILB based interservice communication

**Comment by kishorerj** [comment]
@boredabdel .. Can you please review this

File: services/multi-cluster/ilb/README.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    ```
+
+2. Deploy the two clusters `gke-1` and `gke-2` as specified in [cluster setup](../../../cluster-setup.md)
+
+3. Now follow the steps for cluster registration with Hub and enablement of Multi-cluster Services.
+
+    There are two manifests in this folder:
+
+    - app.yaml is the manifest for the `whereami` Deployment and Service.
+    - ilb.yaml is the manifest for delpoying the whereami service with an Internal load balancer. NOTE: The ip address specified in ilb.yaml should be in the subnet CIDR range of the GKE cluster.
```
Comment (danielmarzini) [inline services/multi-cluster/ilb/README.md:46]: PTAL to type delpoying

File: services/multi-cluster/ilb/README.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+
+5. Now create the service as an ilb with an internal ip address within the CIDR of the subnet of GKE cluster.
+
+    ```bash
+    $ kubectl --context=gke-1 apply -f ilb.yaml
+    service/whereami-ilb created
+    ```
+
+
```
Comment (danielmarzini) [inline services/multi-cluster/ilb/README.md:70]: Please, merge these two files in order to have only one single recipe.

File: services/multi-cluster/ilb/README.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
+# Multi cluster service communication within same network
+
+[Shared VPC](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-shared-vpc#managing_firewall_resources) Shared VPC enables multiple GKE clusters residing across different projects to be part of the same network. This enables different teams to manage their individual projects and communicate using the same shared VPC network centrally managed in the host project. Here we will look at how services residing across different clusters can communicate across cluster boundaries which are part of the same network 
+
+### Relevant documentation
+
+- [Shared VPC Concepts](https://cloud.google.com/kubernetes-engine/docs/concepts/multi-cluster-services)
+- [Setting Up Multi-cluster Services](https://cloud.google.com/kubernetes-engine/docs/how-to/multi-cluster-services)
```
Comment (danielmarzini) [inline services/multi-cluster/ilb/README.md:8]: where are we using MCS in this recipe?

File: services/multi-cluster/ilb/README.md (hunk 2 above)
Comment (kishorerj) [reply services/multi-cluster/ilb/README.md]: done and validated that it works

**Comment by boredabdel** [comment]
I'm busy with a conference this week. @danielmarzini once this is ready to merge let me know :)

**Comment by danielmarzini** [comment]
Hey @kishorerj, I think this one is a dup of [this one](https://github.com/GoogleCloudPlatform/gke-networking-recipes/tree/main/services/single-cluster/internal-lb-service). wdyt?

**Comment by kishorerj** [comment]
> Hey @kishorerj, I think this one is a dup of [this one](https://github.com/GoogleCloudPlatform/gke-networking-recipes/tree/main/services/single-cluster/internal-lb-service). wdyt?

Yes Daniel  you are right, the yamls are same. I didnt notice this. The only difference is the use case and scenario is different. In mine I have explained how this can be used to facilitate multi-cluster communication.

**Review by bowei — APPROVED** [review]
LGTM

**Pull request merged**